<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Font_Settings_Screen</class>
 <widget class="QWidget" name="HNS_Font_Settings_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LBL_FontName1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview5" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview2" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_FontSet_Cancel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName6">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName9">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>460</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview3" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>150</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_FontSet_Back">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview1" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_FontSet_ApplyAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>170</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>APPLY TO WHOLE
MESSAGE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview7" native="true">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>490</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview6" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>320</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview8" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview9" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>490</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FontName8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FONT 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>MESSAGE FONT OPTIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_FontSet_ApplyAfter">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>170</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>APPLY THIS
FONT FORWARD</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_FontSet_Save">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Fonts_Preview4" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
